### DISCUSSION TITLE
Open Ulauncher in specific search mode via terminal

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [Ulauncher/Ulauncher#1374] [Q&A]
Hello,
my setting looks like that:
![image](https://github.com/Ulauncher/Ulauncher/assets/32805117/77d4c12a-89e4-4de0-b634-6305a330cfeb)
wanted to know if it is possible to open up ulauncher from terminal and if we write in it, we directly search in Stack overflow?

**Comment by friday** [comment]
Yeah, if you upgrade to v6 (beta): `gapplication action io.ulauncher.Ulauncher set-query "'goo '"`
